lms/djangoapps/teams/static/teams/js/spec/create_team_spec.js

```diff
@@ -0,0 +1,73 @@
+define([
+    'jquery',
+    'backbone',
+    'common/js/spec_helpers/ajax_helpers',
+    'teams/js/views/create_team'
+], function ($, Backbone, AjaxHelpers, CreateTeamView) {
+    'use strict';
+
+    describe('CreateTeam', function () {
+        var createTeamView,
+            expectContent = function (selector, text) {
+                expect(createTeamView.$(selector).text().trim()).toBe(text);
+            },
+            verifyDropdownData = function (selector, expectedItems) {
+                var options = createTeamView.$(selector)[0].options;
+                var renderedItems = $.map(options, function( elem ) {
+                    return [[elem.value, elem.text]];
+                });
+                for (var i = 0; i < expectedItems.length; i++) {
+                    expect(renderedItems).toContain(expectedItems[i]);
+                }
+            };
+
+        beforeEach(function () {
+            setFixtures('<div class="teams-content"></div>');
+            createTeamView = new CreateTeamView({
+                el: $('.teams-content'),
+                teamParams: {
+                    topicName: 'Awesomeness',
+                    languages: [['a', 'aaa'], ['b', 'bbb']],
+                    countries: [['c', 'ccc'], ['d', 'ddd']]
+                }
+            }).render();
+        });
+
+        it('can render itself correctly', function () {
```

Comment:
Wow! You've really verified every single component of this view. I like that your test case is comprehensive, but I'm concerned that this is going to be a really brittle test case. Can you do a couple things to clean it up?
* We already test the header view in `header_spec.js`, so I think you should only check that the header appears.
* Move the logic that checks form rendering into `edit_team_spec.js`, since it has to do with the edit team view.
    * After you've moved that logic, you can simply here test that the form is present.